### ISSUE TITLE
Pylance shows a tuple of excepcions arent valid exception

### ISSUE THREAD

**Issue by Durgeoble** [microsoft/pylance-release#1216]
detected in the import from ftplib module all_errors.
Original code from ftplib.
`# All exceptions (hopefully) that may be raised here and that aren't`
`# (always) programming errors on our side`
`all_errors = (Error, OSError, EOFError)`

![image](https://user-images.githubusercontent.com/2791206/116718206-04988080-a9da-11eb-856e-82447bff16aa.png)

**Comment by erictraut** [comment]
Can you please provide a self-contained, minimal piece of code that reproduces the behavior? I'm not able to repro the problem from the screen shot above. Here's what I tried, and it seems to work fine:

```python
all_errors = (OSError, EOFError)

def func():
    try:
        ...
    except all_errors as e:
        ...
```

I didn't include `Error` in the tuple because that isn't a standard built-in type, and I don't know how you have defined it.

**Comment by Durgeoble** [comment]
sure
language python 
```
import ftplib  # the library from comes the code above
def ftp_connect(host, user, password, passive=False):
    try:
        logger.debug(host)
        ftp = ftplib.FTP(host, user, password, "", timeout=10)
        ftp.set_pasv(passive)

    except ftplib.all_errors as e:
        logger.exception(f'FTP connection error, cant connect to {host}')
    return ftp
```

**Comment by jakebailey** [comment]
ftplib is a standard library module, and the tuple in question is defined here: https://github.com/python/typeshed/blob/b0c6598fbaccf92f15a8bc9c47934fd6276eaf78/stdlib/ftplib.pyi#L25

**Comment by erictraut** [comment]
OK, thanks for the additional details.

This appears to be a bug in the ftplib.pyi type stub. It defines `all_errors` as follows:
```python
all_errors = Tuple[Type[Exception], ...]
```
In other words, it's defining `all_errors` as a type alias.

Based on the documentation, `all_errors` is not supposed to be a type alias; it's supposed to be a variable with the type `Tuple[Type[Exception, ...]]`. It should therefore be defined this way:
```python
all_errors: Tuple[Type[Exception], ...]
```

I can submit a PR to the typeshed repo with this fix.

**Comment by erictraut** [comment]
Here's the typeshed PR: https://github.com/python/typeshed/pull/5271

**Comment by erictraut** [comment]
@Durgeoble, it's not a bug in the python library, but it is a bug in the typeshed stubs that describe the types used in the python library.

**Comment by erictraut** [comment]
Thanks to very quick turnaround from the maintainers of typeshed, the PR was already accepted and merged. I've pulled the latest typeshed stubs into the pyright repo, so the fix will be incorporated into an upcoming release of pylance.

Thanks again for the bug report.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.5.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202150-5-may-2021
